specification/overview.md

```diff
@@ -26,21 +26,52 @@ This specification is designed to be used by three major groups:
 
 The FOCUS working group will develop an open-source specification for billing data. The schema will define data dimensions, metrics, a set of attributes about billing data, and a common lexicon for describing billing data.
 
-## Design Notes
+## Design Principles
 
 The following principles were considered while building the specification.
 
-### Provider-Neutral Approach by Default
+### FOCUS is an iterative, living specification
+
+* Incremental iterations of the specification released on a regular basis will provide higher value to practitioners and allow feedback as the specification develops. The goal is not to get to a complete, finished specification in one pass.
+
+### Working backwards with ease of adoption
 
-While the schema, naming, terminology and attributes of many providers were reviewed during development, this specification aims to be provider-neutral. In some cases, the approach may closely resemble one or more provider's implementation, while in other cases, the approach might be new. In all cases, the FOCUS group (community composed of FinOps practitioners, Cloud and SaaS providers and FinOps vendors) will attempt to prioritize alignment with the FinOps [Framework][FODOF] and [Capabilities][FODOFC].
+* The FOCUS group working on the specification aims to work backwards from essential FinOps capabilities that practitioners need to perform to prioritize the dimensions, metrics and the attributes of the cost and usage data that should be defined in the specification to fulfill that capability.
+* Be FinOps scenario-driven. Define columns that answer scenario questions; don't look for scenarios to fit a column, each column must have a use case.
+* Don't add dimensions or metrics to the specification just because it can be added.
+* When defining the specification, consideration should be made to existing data already in the major providers' (AWS, GCP, Azure, OCI) data sources.
+* As long as it solves the FinOps use case, there should be a preference to align with data that is already present in a majority of the major providers.
```

Comment:
I just worry that saying this at the top of the document does two things:
- Makes it harder for us to take this to other providers and other types of providers / cost sources (saas, on-prem) and make the justification that FOCUS is built with the general FinOps needs in mind and not try to force a very cloud specific taxonomy on non-cloud providers.
- Confuses from the working backwards approach we detailed up-top. It would be logical for someone to ask which is it? The reality is that common FinOps capabilities are mostly based on well known CSP cost and usage data sets already. However, if there happens to be something that practitioners really need to do, that major providers aren't enabling today, that's still something that should be prioritized based on the merit of the the use case not based on if majority of the providers have a way to enable this or not. 

Otherwise FOCUS always ends up being the follower that comes in and standardizes the lowest common denominator not the avenue for the FinOps practitioners to drive the requirements for common FInOps needs

Reply:
Related: upon reading through this second draft, the only big thing that sticks out to me is the contradiction across sections of FOCUS being both provider-neutral and giving preference to major providers.